cse/src/main/java/com/adyen/checkout/cse/UnencryptedCard.java

```diff
@@ -209,11 +196,5 @@ private String removeWhiteSpaces(String string) {
         private String trimAndRemoveMultipleWhiteSpaces(String string) {
             return string != null ? string.trim().replaceAll("\\s{2,}", " ") : null;
         }
-
-        private void require(boolean condition, String message) throws IllegalStateException {
```

Comment:
what happens when on of these values is null? If the encryption still works then this is fine.

Reply:
Yeah it used to ignore validation if something is null, now it ignores validation altogether.